week-2/implement/tally.js

```diff
@@ -0,0 +1,21 @@
+function tally(arr) {
+  const counter = {};
+
+  if (typeof arr === "string") {
+    return "Error, you should pass an array of values";
+  }
+
+  arr.forEach((item) => {
+    if (counter[item]) {
+      counter[item] += 1;
+    } else {
+      counter[item] = 1;
+    }
+  });
+  return counter;
+}
```

Comment:
A nice start for the implementation of the function `tally()`.

I just want to ask if on line 4, why are you checking only the case where the input was a string, is string the only invalid variable type for this function? What about a number?

Reply:
Yes I should actually include a Number and a Boolean to this!